src/EnvironmentSetCommand.php

```diff
@@ -48,7 +49,7 @@ public function handle()
         $contents = file_get_contents($envFilePath);
 
         if ($oldValue = $this->getOldValue($contents, $key)) {
-            $contents = str_replace("{$key}={$oldValue}", "{$key}={$value}", $contents);
+            $contents = Str::replace("{$key}={$oldValue}", "{$key}={$value}", $contents);
```

Comment:
`str_replace` is a vanilla PHP function, not a legacy Laravel helper. In fact, there isn't a `replace` method on Laravel's Str class. Could this line be undone?